tutorials/cloudsdk-js-generator/cloudsdk-js-generator.md

```diff
@@ -13,6 +13,7 @@ primary_tag: products>sap-cloud-sdk
 > This tutorial is not actively maintained and might be partially outdated.
 > Always up-to-date documentation is published on our [documentation portal](https://sap.github.io/cloud-sdk/).
 > We will provide a link to the updated version of this tutorial as soon as we release it.
```

Comment:
Do you want to link in this tutorial to the updated version of this tutorial?

Reply:
I'm not able to answer, as the plan is not clear.
Also, this was not added by the current PR.